msal/tests/Test.MSAL.NET.Unit.net45/Integration/ApplicationGrantIntegrationTest.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.Identity.Client;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Test.Microsoft.Identity.LabInfrastructure;
+
+namespace Test.MSAL.NET.Unit.net45.Integration
+{
+    [TestClass]
+    public class ApplicationGrantIntegrationTest
+    {
+        public const string Authority = "";
+        public const string ClientId = "";
+        public const string RedirectUri = "http://localhost";
+        public string[] MsalScopes = { "https://graph.microsoft.com/.default" };
```

Comment:
We settled on using _ for private member naming -> _password

Reply:
ok, will change